iogt/styles/iogt/modules/_m-profiles.scss

```diff
@@ -68,3 +68,10 @@ label.profiles__terms {
   color: red;
   font-size: $font-small;
 }
+.required:after {
```

Comment:
Is this going to work with feature phones?

Reply:
yes, been using pseudo elements on most of our molo projects, no issues yet, its widely supported on older browsers and most importantly all versions of opera mini, https://caniuse.com/#feat=css-gencontent

![screen shot 2017-10-20 at 14 28 35](https://user-images.githubusercontent.com/9653693/31820981-bf79d0ae-b5a3-11e7-963f-bc5ae0e406d7.png)

Reply:
Cool - thanks for the explanation :)